package.json

```diff
@@ -6,9 +6,14 @@
     "author": "",
     "license": "ISC",
     "dependencies": {
-
+        "lodash": "^3.10.0"
```

Comment:
lodash isnt semver compatible, so you may want to consider more strict reference to 3.10.x or something